src/app-layout/visual-refresh-toolbar/toolbar/trigger-button/index.tsx

```diff
@@ -44,11 +75,145 @@ function TriggerButton(
     badge,
     selected = false,
     highContrastHeader,
+    hasTooltip = false,
+    tooltipText,
+    hasOpenDrawer = false,
+    isMobile = false,
+    isForPreviousActiveDrawer = false,
+    isForSplitPanel = false,
   }: TriggerButtonProps,
   ref: React.Ref<ButtonProps.Ref>
 ) {
+  const containerRef = useRef<HTMLDivElement | null>(null);
+  const tooltipValue = tooltipText ? tooltipText : ariaLabel ? ariaLabel : '';
+  const [showTooltip, setShowTooltip] = useState<boolean>(false);
```

Comment:
What about calling this one something like `[isFocused, setIsFocused]`?

Reply:
tooltip shows when the button is not focused, such as when hovered